node/commons/compatibility-layer.ts

```diff
@@ -132,6 +132,12 @@ export const convertBiggyProduct = async (
         return attribute.labelValue
       })
     })
+
+    product.textAttributes.filter((attribute) => attribute.labelKey === "productClusterNames").forEach((attribute) => {
+      if (attribute.valueId) {
+        convertedProduct.productClusters[attribute.valueId] = attribute.labelValue
```

Comment:
`productClusters` is an array and here is being set as an object. 

Am I missing something? Is this object being converted to an array later?

GraphQL type:
https://github.com/vtex-apps/search-graphql/blob/master/graphql/types/Product.graphql#L30

Reply:
Yeah, actually there it is an object, it gets turned into an array here: https://github.com/vtex-apps/search-resolver/blob/feature/comaptibility-product-clusters/node/resolvers/search/product.ts#L186